http_service/models.py

```diff
@@ -109,7 +109,8 @@ def classify_bug(
 
     model = get_model(model_name)
 
-    model_extra_data = model.get_extra_data()
+    # model_extra_data = model.get_extra_data()
+    model_extra_data = {}
```

Comment:
Why this change?

Reply:
Unrelated change, checking that everything is working as it should without it

Reply:
Everything is good